### ISSUE TITLE
Possible to have multiple series on a time chart?

### ISSUE THREAD

**Issue by simongis** [Esri/cedar#275]
Likely a rookie error, but is it possible to have multiple lines on a line chart?

Sample app [here](http://s3-ap-southeast-2.amazonaws.com/languagesofaustralia.com/SimonJacksonFiles/cedar_hub8.html)

The request seems to be returning the[ expected results](https://services1.arcgis.com/vHnIGBHHqDR6y0CR/ArcGIS/rest/services/CrashStats_2017/FeatureServer/0/query?where=1%3D1&returnGeometry=false&returnDistinctValues=false&returnIdsOnly=false&returnCountOnly=false&outFields=*&sqlFormat=standard&f=json&orderByFields=accident_date&groupByFieldsForStatistics=accident_date&outStatistics=%5B%7B%22statisticType%22%3A%22sum%22%2C%22onStatisticField%22%3A%22bicyclist%22%2C%22outStatisticFieldName%22%3A%22bicyclist_SUM%22%7D%2C%7B%22statisticType%22%3A%22sum%22%2C%22onStatisticField%22%3A%22pedestrian%22%2C%22outStatisticFieldName%22%3A%22pedestrian_SUM%22%7D%5D) from the service, but I must be missing something in parsing the results into the chart.  

```
var chart = new Cedar({"type":"time"});

  var datasets = [
    {
      "url": "https://services1.arcgis.com/vHnIGBHHqDR6y0CR/ArcGIS/rest/services/CrashStats_2017/FeatureServer/0",
      "query": {
        "orderByFields": "accident_date",
		"groupByFieldsForStatistics": "accident_date",
		"outStatistics": [
		{
		"statisticType": "sum",
		"onStatisticField": "bicyclist",
		"outStatisticFieldName": "bicyclist_SUM"	  
		},
		{
		"statisticType": "sum",
		"onStatisticField": "pedestrian",
		"outStatisticFieldName": "pedestrian_SUM"	  
		}
		]
      }
    }
  ];
  
  var series = [
    {
      "category": { "field": "accident_date", "label": "Date" },
      "value": { "field":"bicyclist_SUM","label":"Bikes" }
    },
	{
      "category": { "field": "accident_date", "label": "Date" },
      "value": { "field":"pedestrian_SUM","label":"Peds" }
    }
	
  ];

  chart.datasets = datasets;
  chart.series = series;
chart.show({
  elementId: "#chart",
  autolabels: true
});
```

**Comment by tomwayson** [comment]
@simongis no, not a rookie error. This is coming soon in v1, working on it now actually. What's confusing is that in the transition period we went ahead an updated the JSON API to take multiple series, but we're ignoring all but the first.

I will report back on this issue when the beta is ready for testing.

**Comment by simongis** [comment]
Thanks @tomwayson.  Whats the ETA for v1?  Assume it would be after our conference in a couple of weeks?

**Comment by tomwayson** [comment]
Working to have a publicly available beta by the end of this week.

**Comment by simongis** [comment]
That's great.  Can you respond on this issue when it is available and I will add an iFrame to the [prototype site](http://vicroads-esriau.opendata.arcgis.com/pages/towards-zero) we will be showing.

**Comment by tomwayson** [comment]
Of course. The site's looking good!

**Comment by tomwayson** [comment]
FYI - you can track progress on our beta here: https://github.com/Esri/cedar/labels/v1-beta

Unfortunately that list keeps growing instead of shrinking. Looks like I was overly optimistic to think that it would be available by the end of the week. 😞

**Comment by tomwayson** [comment]
So yeah, over 2 months later, an alpha release is finally available: https://github.com/Esri/cedar/releases/tag/v1.0.0-alpha
